### PR TITLE
Precedent input when typing ↑

### PR DESCRIPTION
**Pull request by rchoffardet**
As suggested here #5 , here is the PR :)

### CODE REVIEW

File: resources/assets/js/app.js [hunk 1]
```diff
@@ -25,6 +28,19 @@ window.addEventListener('click', event => {
     }, 200);
 });
 
+input.addEventListener('keydown', event => {
+    if (event.keyCode === 38) {
+        input.value = history.getPrevious();
+    } else if (event.keyCode === 40) {
```
Comment (freekmurze) [inline resources/assets/js/app.js]: Could you avoid `else` by making using of early returns?

File: resources/assets/js/History.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    getNext() {
+        if (this.index > 0) {
+            return this.items[--this.index];
+        }
+
+        return '';
+    }
+}
+
+export default History;
```
Comment (freekmurze) [inline resources/assets/js/History.js]: Make this file end with a newline.

**Comment by rchoffardet** [comment]
It should be good

**Comment by rchoffardet** [comment]
Just add a limit to the stored history so they won't have any problems in the future.

**Comment by freekmurze** [comment]
Closing this as it's already included in #20

**Pull request closed without merging**